tests/test_core.py

```diff
@@ -6428,6 +6428,22 @@ def test_asm_pgo(self):
     shutil.move(self.in_dir('src.cpp.o.js'), self.in_dir('pgoed2.js'))
     assert open('pgoed.js').read() == open('pgoed2.js').read()
 
+  def test_response_file(self):
+    with open('rsp_file', 'w') as f:
+      f.write('-o %s/response_file.o.js %s' % (self.get_dir(), path_from_root('tests', 'hello_world.cpp')))
+    subprocess.check_call([PYTHON, EMCC, "@rsp_file"])
+    self.do_run('' , 'hello, world', basename='response_file', no_build=True)
+
+  def test_linker_response_file(self):
+    objfile = os.path.join(self.get_dir(), 'response_file.o')
+    subprocess.check_call([PYTHON, EMCC, '-c', path_from_root('tests', 'hello_world.cpp'), '-o', objfile])
+    # TODO(sbc): This should expand into -Wl,foobar which is currently ignored
```

Comment:
Is this test just testing that these files can be parsed?

Reply:
Yes.. currently -Wl,xxx flags are always ignore because the clang driver never runs the linker.